web/src/store/index.js

```diff
@@ -171,18 +172,29 @@ const mutations = {
         normalization_argument: data.normalization_argument,
         transformation: data.transformation,
         scaling: data.scaling,
+
+        // imputed measurements
+        measurement_table,
+
+        // reset
+        validatedMeasurements: null,
+        validatedGroups: null,
+        validatedMeasurementsMetaData: null,
+        validatedSampleMetaData: null,
```

Comment:
Same comment as in #260 : please include these in the defaults and include `measurement_table` as well.

I'll just approve this so you can do it on either branch.